quickstack/manifests/backups.pp

```diff
@@ -0,0 +1,102 @@
+class backups (
+        $user		= "backup",
+	$script_src	= "puppet:///modules/quickstack/compute_backup.sh",
+	$backups_dir    = '/backups',
+        $log_file       = '/var/log/backups.log', 
+        $ssh_key        = "puppet:///modules/quickstack/backup_key.pub",
+        $cron_email     = 'moc-kaizen-l@bu.edu',
```

Comment:
This is currently overridden by hiera to be kamfonik@bu.edu... I have some diagnostics in place that I would like to keep up for the first week or so that we have this going in production.  But I don't really need to spam the whole list.  There will be an additional pull request once I turn off the diagnostic emails, and at that point I'll change the hiera file to point to moc-kaizen-l as well.